package/MDAnalysis/exceptions.py

```diff
@@ -35,6 +35,26 @@ class SelectionError(Exception):
 class NoDataError(ValueError, AttributeError):
     """Raised when empty input is not allowed or required data are missing.
 
+    This exception is used in various scenarios:
+    -   Raised when a Topology Attribute (e.g., bonds, charges) is not
```

Comment:
```suggestion
    * Raised when a :class:`~MDAnalysis.core.topologyattrs.TopologyAttr` (e.g., bonds, charges) is not
```

A topology attribute in this case is not just a concept but also a specific class. To briefly explain the syntax, the `:class:` syntax links to the class in the rendered docs and the `~` means that only the `TopologyAttr` part gets rendered, so it looks neater. The `*` is the syntax to use for unordered lists in Sphinx.

Reply:
- It will look like this in the IDE.
![image](https://github.com/MDAnalysis/mdanalysis/assets/91054457/a2ca650c-030c-4a37-a40e-cfa13f1d4eb2)
- Should it be like that ?

Reply:
Sure, I'm not sure IDE representation has been a high priority for MDAnalysis in the past. It should look nicer when rendered if you make the change.